src/cmd-build

```diff
@@ -242,9 +242,19 @@ img_base=tmp/${imageprefix}-base.qcow2
 # forgive me for this sin
 checksum_location=$(find /usr/lib/coreos-assembler-anaconda/ -name '*CHECKSUM' | head -1)
 
+build_image() {
+	local size kargs
+	size="$(python3 -c 'import sys, yaml; print(yaml.load(sys.stdin)["size"])' < "$configdir/image.yaml")G"
+	kargs="$(python3 -c 'import sys, yaml; args = yaml.load(sys.stdin)["extra-kargs"]; print(" ".join(args))' < "$configdir/image.yaml")"
+	kargs="$kargs console=${VM_TERMINAL} console=${VM_TERMINAL},115200n8"
```

Comment:
Hmm, looks like we are specifying the same console arg twice? I think we want something like this instead: `console=tty0 console=${VM_TERMINAL}`, right?

Reply:
I don't know exactly, so I'll defer to you.